app/controllers/request-psp-test-account/get.controller.js

```diff
@@ -0,0 +1,22 @@
+'use strict'
+
+const { response, renderErrorView } = require('../../utils/response')
+const { NOT_STARTED, CREATED, REQUEST_SUBMITTED } = require('../../models/psp-test-account-stage')
+const goLiveStage = require('../../models/go-live-stage')
+
+module.exports = async (req, res) => {
+  const service = req.service
+  service.currentPspTestAccountStage = CREATED
```

Comment:
Does this intentionally set the `currentPspTestAccountStage` and then do logical comparisons on it later? Assuming this value can change should this only be setting it if there isn't already an existing value?

Reply:
🤦🏽 This should haven't been in.